desc/compute/_qs.py

```diff
@@ -1,16 +1,16 @@
-"""Compute functions for quasisymmetry objectives, ie Boozer, Two-Term, and Triple Product."""
+"""Compute functions for quasisymmetry objectives."""
 
 from desc.backend import jnp, put, sign
-
-from ._core import check_derivs
+from .utils import check_derivs
 from ._field import (
     compute_magnetic_field_magnitude,
     compute_covariant_magnetic_field,
     compute_B_dot_gradB,
+    compute_boozer_magnetic_field,
 )
 
 
-def compute_boozer_coords(
+def compute_boozer_coordinates(
```

Comment:
Do we want to rename other `compute_X_coords` functions too?

Reply:
Probably, but what do others think? @f0uriest @dpanici 

I made this change because it seemed like we have been spelling out all the other compute function names with verbose language.

Reply:
My vote would be yes, I like it being more verbose. Maybe is not necessary to change during this PR though